doc/ref_model/chapters/chapter05.md

```diff
@@ -51,7 +51,7 @@ The following sections detail the NFVI SW profile features per type of virtual r
 |------------------|----------------|----------------|------------------------------------------------------------------------------------------------|
 | nfvi.com.cfg.001 | CPU allocation ratio  | Value | Number of virtual cores per physical core  |
 | nfvi.com.cfg.002 | NUMA awareness | Yes/No  | Support of NUMA at the virtualization layer  |
-| nfvi.com.cfg.003 | CPU pinning capability  | Yes/No | Binding of a process to a dedicated CPU |
+| nfvi.com.cfg.003 | CPU pinning capability  | Yes/No | Binding a vCPU to either a physical core on a non-SMT architecture or a core thread on a SMT architecture  |
```

Comment:
@karinesevilla @deebhatia Hi, The Issue reported says that the line item description doesn't take into account cores and SMT. However, the change additionally replaces "process" with "vCPU". While technically correct, IMHO that change makes it harder to understand because what the reader cares about is binding the process to the h/w. The fact that the process is bound to a vCPU which is in-turn bound to the h/w adds a level of complexity that doesn't enhance the understanding, and may actually confuse some people. **Can we revert vCPU back to process?**

Also, **can we shorten the physical description** to something more concise, such as "_Binds a process to a physical core or thread_"? My thinking is, if we have to teach the difference between cores, threads, SMP and SMT, the reader probably needs to brush up on more than we can explain in one sentence. Thanks, -Mark

Reply:
@markshostak Hi Mark, the intention was to precise the definition and for this reason I am not aligned with the simplication you are proposing. 
vCPU is used in chapter 3 & 4 and more accurate than process is this case. Adding the SMT architecture in the defintion aims to explain why the binding is with physical core or  thread. As SMT wasn't used before , a note can be added to develop the acronym if needed.

Reply:
@karinesevilla Bonjour Karine, I understand. 
To your point, we probably want to add a definition to the glossary, rather than define terms in a table of capabilities. The vCPU definition should explain about physical core and thread in SMT. How about "_Binds a process/vCPU to a physical core or SMT thread_"?

Reply:
@markshostak, @deebhatia 
I agree on Mark's compromise

Reply:
Yeah. Make sense.

Reply:
has this been resolved?

Reply:
@markshostak @rabi-abdel Done the description change on behalf of @karinesevilla .